core/modules/user/css/user.css

```diff
@@ -201,3 +201,46 @@ form:not(.user-admin-settings) .password-strength-help-text.strong em {
 .profile dd {
   margin: 0 0 1em 0;
 }
+.user-simplified-page .user-simplified-page-logo {
+  max-width: 150px;
+}
+/* Maintenance page styles */
+body.user-simplified-page-body {
+  padding: 1em;
+}
+.user-simplified-page-wrapper {
+  padding: 2em;
+  max-width: 770px;
+  min-width: 255px;
+  margin: 0 auto;
+  background: white;
+  position: relative;
+}
+@media all and (min-width: 768px) {
+  body.user-simplified-page-body  {
+    width: 100%;
+    display: table;
```

Comment:
I'm no CSS expert, but at this point I think it might be better (or at least more expected) to use flexbox or grids rather than `display: table` for vertical centering.

See the "Can you use grid" and "Can you use flexbox" answers (the last two) from https://css-tricks.com/centering-css-complete-guide/

Reply:
Oh I see, you're matching the existing CSS from `seven/css/style.css`.

Maybe we should split this out into a dedicated CSS file, like `user-simplified-page.css`, and then have the Seven theme exclude it in the .info file? That would allow this CSS to be more easily override or ignored by admin themes or custom themes.

Reply:
> Oh I see, you're matching the existing CSS from seven/css/style.css.

Yes, and therefore, I thought they should be similar, and paired with the maintenance page CSS, as they are now, so its easier to keep them consistent.
> Maybe we should split this out into a dedicated CSS file...that would allow this CSS to be more easily override or ignored 

Just to be clear, you mean that Seven would then ignore that file (by referencing `css/user-simplified-page.css` in the `seven.info` but *not* providing that file), then add the CSS that *should* be in that file, directly into Seven's `style.css`?

Reply:
> Just to be clear

Yes, exactly as you outlined is what I was thinking. That way you don't have Seven fighting with the default CSS.

Reply:
OK, done